OpenClaw/Engine/Resource/Miniz.h

```diff
@@ -237,7 +237,7 @@ extern "C" {
     typedef uLong uLongf;
     typedef void *voidp;
     typedef void *const voidpc;
-#define Z_NULL                0
+#define Z_nullptr                0
```

Comment:
This is one of the reasons I don't like system/project wide search-and-replace in already working (and not maintained) codebase :)

Reply:
Sorry for long response time, it should be fixed now.